README.md

```diff
@@ -1 +1,24 @@
-# actions
\ No newline at end of file
+## GitHub Actions Maintainer 
```

Comment:
What does "Maintainer" mean here?

Reply:
I asked myself a question "what this repository is all about"? To me it is a set of GitHub Actions that help maintainers to maintain `fastlane` project.  That's why I called it `GitHub Actions Maintainer`. Any suggestions how to improve this name @janpio ?

Reply:
"Fastlane GitHub Actions" ;)

Reply:
"A set of GitHub Actions fastlane uses."